### PR TITLE
Portfolio page

### PR DESCRIPTION
**Pull request by LEvinson2504**
Im not sure the way I passed the className props is what we wanted, please have a look.

### CODE REVIEW

File: public/client/components/PageHeader.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const PageHeader = ({ title, subtitle, className }) => {
+  return (
+    <div className={`${className} || container sm:m-3 sm:p-4`}>
```
Comment (JanuaryLimes) [inline public/client/components/PageHeader.js:5]: imho `||` doesn't make sense here

Comment (JanuaryLimes) [reply public/client/components/PageHeader.js:5]: ok I see now what you tried to do but this `<div className={`${className || "grid..."}`}>` will work differently, in this line you have to add additional quotation marks like `"container sm:m-3 sm:p-4"`

Comment (zoltot) [reply public/client/components/PageHeader.js]: @LEvinson2504  Yeah, its important to remember in javascript everything between `` that isn't wrapped with ${} will be a regular string

**Review by zoltot — APPROVED** [review]

File: public/client/components/PageHeader.js (hunk 1 above)
Comment (LEvinson2504) [reply public/client/components/PageHeader.js]: @JanuaryLimes @zoltot I see

**Pull request merged**